spec.md

```diff
@@ -259,14 +259,25 @@ will need to provide this identification information to the Service Broker on
 each request. Platforms MAY support this feature, and if they do, they MUST
 adhere to the following:
 - For any OSBAPI request that is the result of an action taken by a Platform's
-  user, there MUST be an associated `X-Broker-API-Originating-Identity` header on that HTTP
-  request.
+  user, there MUST be an associated `X-Broker-API-Originating-Identity` header
+  on that HTTP request.
 - Any OSBAPI request that is not associated with an action from a Platform's
   user, such as the Platform refetching the catalog, MAY exclude the header from
   that HTTP request.
-- If present on a request, the `X-Broker-API-Originating-Identity` header MUST contain the
-  identify information for the Platform's user that took the action to cause the
-  request to be sent.
+- If present on a request, the `X-Broker-API-Originating-Identity` header MUST
+  contain the identify information for the Platform's user that took the
+  action to cause the request to be sent.
+
+If the Platform chooses to group multiple end-user operations into one request
+to the Broker, then the identity information associated with that one request
+MUST accurately reflect the desired indentity associated for each individual
+change.
+For example, if two different users modify a Service Instance, but the Platform
+sends just one PATCH request to the Broker, the Broker would associate both
+sets of changes with just one user. Whether it is correct or not for that
```

Comment:
> Whether it is correct or not for that
> one user to be associated with both sets of changes (for auditing or
> authorization purposes) is out of scope of this specification,

I would drop the entire second paragraph.

Reply:
Do you mean everything after "For example", or everything after "Wheter it is correct"?